packages/charts/package.json

```diff
@@ -3,12 +3,13 @@
     "version": "1.0.0",
     "description": "Charts for RedHat Cloud Services project.",
     "main": "dist/index.js",
```

Comment:
You should update this to point at `/dist/cjs`

Reply:
Not really, this deliberately points to UMD since it's the fallback way, less breaking changes. If folks want to use `cjs` they can do that by importing said file.

Reply:
Ok then we are definitely missing the files attribute to include the cjs files..

Reply:
I have noticed that you have changed the main in one of the packages though. Was that deliberate or only for testing purposes?

Reply:
Also, if we are releasing v2, I would be in favor using the cjs over UMD anyway. Current users will still be able to use UMD but we will point them in the right direction for the future.

Reply:
Yeah, that was deliberate, because it was poiting to wrong file.